test_requirements.txt

```diff
@@ -5,4 +5,4 @@ coverage>=3.7
 pytest-cov>=1.8.1
 coveralls>=0.5
 python-coveralls>=2.5
-influxdb>=5.3.0
\ No newline at end of file
+influxdb-client
```

Comment:
Should we specify version constraints here? I usually update this file using the output of `pip freeze`.